config/docker/thinking_sphinx.yml

```diff
@@ -1,44 +1,43 @@
-common: &sphinx
-  configuration_file: <%= ENV.fetch('THINKING_SPHINX_CONFIGURATION_FILE') { Rails.root.join("config/#{Rails.env}.sphinx.conf")} %>
+production:
+  # ----------------------------------
+  # Thinking Sphinx configuration
+  # see https://gist.github.com/pat/a7d73376dd657b4457092efc9e9c418a
+  # ----------------------------------
+  mysql41: 9306
+  address: <%= ENV['THINKING_SPHINX_ADDRESS'] || '0.0.0.0' %>
+
   big_document_ids: true
+
+  configuration_file: <%= ENV.fetch('THINKING_SPHINX_CONFIGURATION_FILE') { Rails.root.join("config/#{Rails.env}.sphinx.conf")} %>
```

Comment:
this seems also like daemon config, but I guess needed also to generate it in the first place...

Reply:
Yes, thinking sphinx will generate the .conf file and place it to the location specified here.

Reply:
My comment was about grouping of settings. No problem with the value :)